samples/app/FilamentApp.cpp

```diff
@@ -677,7 +677,7 @@ void FilamentApp::Window::configureCamerasForWindow() {
 FilamentApp::CView::CView(Renderer& renderer, std::string name)
         : engine(*renderer.getEngine()), mName(name) {
     view = engine.createView();
-    view->setClearColor({ 0 });
+    view->setClearColor(LinearColorA{ 0 });
```

Comment:
These are bound to happen again -- the code is legal AFAIK. In fact, it's even possible that some Linters will complain.

Reply:
Do you want me to change anything for this?